datafusion/functions-aggregate/src/first_last.rs

```diff
@@ -79,15 +79,7 @@ impl Default for FirstValue {
 impl FirstValue {
     pub fn new() -> Self {
         Self {
-            signature: Signature::one_of(
-                vec![
-                    // TODO: we can introduce more strict signature that only numeric of array types are allowed
-                    TypeSignature::ArraySignature(ArrayFunctionSignature::Array),
-                    TypeSignature::Numeric(1),
-                    TypeSignature::Uniform(1, vec![DataType::Utf8]),
-                ],
-                Volatility::Immutable,
-            ),
+            signature: Signature::any(1, Volatility::Immutable),
```

Comment:
Ideally first/last value get the first/last item in the column, so it should be the type of the column.

I forgot why we don't use `Any`.
Without the change, we need to add another type signature for boolean, so I change it to `Any` in this PR